ui/admin/app/components/form/credential-library/vault-generic/index.hbs

```diff
@@ -169,14 +169,21 @@
       </:fieldset>
 
       <:field as |F|>
-        <F.SelectText
+        <F.KeyValue
           @name='credential_mapping_overrides'
           @selectOptions={{this.mappingOverrides}}
           @model={{@model}}
           @disabled={{form.disabled}}
           @removeDuplicates={{true}}
           @width='400px'
```

Comment:
None blocking question: I know the value was set already, but I think this value should be `100%`.

Screenshot at `400px`:
![Screenshot 2024-03-28 at 8 30 35 AM](https://github.com/hashicorp/boundary-ui/assets/9775006/61b46783-fa1c-497b-af26-2b91ec1675c6)

Screenshot at `100%`:
![Screenshot 2024-03-28 at 8 31 13 AM](https://github.com/hashicorp/boundary-ui/assets/9775006/326ac649-53f3-4b39-b52c-67b37173c6d5)

Reply:
thanks for the catch, fixed